src/Components/Card.js

```diff
@@ -6,14 +6,28 @@ class Card extends Component {
   }
 
   render() {
+    const displayCard = this.props.parades.filter((parade) => {
+     
+      console.log(this.props.toggle, 'heyy')
+      return parade.id == this.props.id
+    })
+    
     return(
       <div>
-        <article>{this.props.name}</article>
+        <button onClick={this.props.toggle}>X</button>
```

Comment:
This button works functionally, but it would be great to have it on the card visually.